apps/emqx/include/logger.hrl

```diff
@@ -61,7 +61,9 @@
     )
 end).
 
--define(AUDIT(_Level_, _Msg_, _Meta_), begin
+-define(AUDIT(_Level_, _From_, _Meta_), ?AUDIT(_Level_, _From_, undefined, _Meta_)).
```

Comment:
It's a bit confusing that this "shortcut" version of the macro translates the omitted argument to Msg rather than Meta. This is quite unexpected for anyone who used OTP logger macro, where Meta is the optional argument and Msg is the mandatory one.